### PR TITLE
docs(wb): corrections and clarifications to guide

### LINKED ISSUE #502: Weight and balance documentation
A couple of minor comments on the latest weight and balance documentation:

In the section on changing default weights using simBrief, the screenshot of the "my fleet" page looks different than mine does when I've downloaded the latest stable and dev/exp simBrief profiles:
From the doc:
Screenshot (2025)
From the website:
Screenshot (2023)
Note the "Comments" field. The current stable profile has the word "Stable" at the end of the line, so it doesn't show up in the "My Fleets" view.  The latest dev/exp profile does not have dev or exp in the comments field at all. This makes sense since the ability to change the default weights does not apply to the stable profile. I suggest using a new screenshot with only the dev/exp simBrief profile present in the "My Fleets" view.

In the same section, after it says, "Change the default weights as required," it needs to say, "Select 'Save Aircraft'." 

In the section on "Do Not Overload the Aircraft" -- where it says that simBrief automatically reduces the number of passengers and/or bags to comply with the weight restrictions, it should probably also point out that although the actual number of passengers and bags loaded onto the airplane may be reduced, if the user had selected a specific number of passengers, the OFP will still reflect the user-selected number of passengers.  (Ref: https://discord.com/channels/738864299392630914/805556372904476672/945578258966151189)

### PR DESCRIPTION
**Pull request by Valastiri**
closes: #502 

## Summary
Corrects and clarifies the following points:
- Updates image to only have one sample airframe in the fleet. (Stable airframe was saved by Val on simbrief and does n to have a "leading STABLE" in front of it)
- Tells user to "save aircraft" after any modifications to the frame.
- Adds the clarification to do not overload section

### Location
- docs/fbw-a32nx/feature-guides/loading-fuel-weight.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/loading-fuel-weight.md [hunk 1]
```diff
@@ -129,15 +129,15 @@ These default weights cannot be changed in the aircraft at the moment. It is pla
...
+To do this open your simBrief airframe on the simBrief website under "MY FLEET" by clicking the Edit button.
 
 ![SimBrief MyFleet Page](../assets/feature-guides/simbrief/simbrief-myfleet.png "SimBrief MyFleet Page")
 
 Scroll down to find the entries for Passenger Weight and Baggage Weight.
 
 ![SimBrief Pax and Baggage Weights](../assets/feature-guides/simbrief/simbrief-pax-baggage.png "SimBrief Pax and Baggage Weights")
 
-Change the default weights as required. The A32NX will use these new values when you import an operational flight plan based on this airframe into aircraft the next time.
+Change the default weights as required and make sure you select Save Aircraft at the bottom of the page. The A32NX will use these new values when you import an operational flight plan based on this airframe into aircraft the next time.
```
Comment (donstim) [inline docs/fbw-a32nx/feature-guides/loading-fuel-weight.md]: Looks like you left out a "the," "based on this airframe into **the** aircraft the next time. Or, how about, "The A32NX will use these new values the next time you import an operational flight plan based on this airframe."?

File: docs/fbw-a32nx/feature-guides/loading-fuel-weight.md [hunk 2]
```diff
@@ -151,6 +151,9 @@ Change the default weights as required. The A32NX will use these new values when
         - MLW - empty weight - (all reserve/contingency fuel + extra fuel) , or 67400 - 42500 - (all reserve fuel/contingency/extra fuel).
     - When manually loading the payload, the MLW and MTOW limits are currently not checked and are the responsibility of the pilot.
     - Simbrief does manage payload limits by reducing the number of bags or passengers automatically. This is picked up by the aircraft correctly then.
+    
+    !!! tip ""
+        If you manually specify a custom amount of passengers the OFP will still reflect the user-selected number of passengers. SimBrief identifies this as "booked passengers" and it will appear unmodified on the OFP.
```
Comment (donstim) [inline docs/fbw-a32nx/feature-guides/loading-fuel-weight.md]: Maybe "If you manually specify a custom **number** of passengers, and this number needs to be reduced to comply with weight limitations, the OFP will still reflect the user-selected number of passengers."?

Comment (Valastiri) [reply docs/fbw-a32nx/feature-guides/loading-fuel-weight.md]: Hmm maybe I misunderstood what the navigraph folks meant. 

Maybe we make it a little longer: 

If you manually specify a custom number of passengers the OFP will still reflect the user-selected number of passengers. 
If the number of passengers you have chosen needs to be reduced to comply with weight limitations this is not reflected on the simBrief OFP. At the time when the OFP was generated, it interprets the passenger count as "booked passengers" and not current passengers on board the aircraft.

File: docs/fbw-a32nx/feature-guides/loading-fuel-weight.md (hunk 1 above)
Comment (Valastiri) [reply docs/fbw-a32nx/feature-guides/loading-fuel-weight.md]: Will use this suggestion verbatim

File: docs/fbw-a32nx/feature-guides/loading-fuel-weight.md (hunk 2 above)
Comment (Valastiri) [reply docs/fbw-a32nx/feature-guides/loading-fuel-weight.md]: If you manually specify a custom number of passengers that exceeds aircraft weight limitations please be aware that the OFP will still reflect the user-selected number of passengers.

This is in part due to the fact that at the time when the OFP was generated, it interprets the passenger count as "booked passengers" and not current passengers on board the aircraft.

---

Went with this instead.

Comment (donstim) [reply docs/fbw-a32nx/feature-guides/loading-fuel-weight.md]: LGTM

**Comment by Valastiri** [comment]
Thanks again!

**Pull request merged**